src/gsAssembler/gsAssemblerOptions.h

```diff
@@ -272,8 +272,8 @@ struct gsAssemblerOptions
     // If set to a value different than zero, it controls the
     // allocation of the sparse matrix, ie. the maximum number of
     // non-zero entries per column (set to: A * p + B)
-    double bdA;
-    int    bdB;
+    double  bdA;
+    index_t bdB;
```

Comment:
This should be either int or short_t. I would prefer int.
 It is not related to index_t.